tests/utils/stl/util.py

```diff
@@ -77,8 +77,9 @@ def decodeOutput(bytes):
     try:
         return bytes.decode()
     except UnicodeError:
+        # Use 'backslashreplace' to avoid throwing another exception for unrecognized characters.

```

Comment:
I'm not fond of "unrecognized characters" which suggests that the transcoder doesn't understand the source character set; the issue here is that tests sometimes emit byte sequences that aren't valid encoded characters. I'd prefer something like "when tests emit garbage bytes."

This is absolutely not worth resetting testing.